spec/bolt/inventory/inventory2_spec.rb

```diff
@@ -682,6 +687,329 @@ def common_data(transport)
         end
       end
     end
+
+    describe 'get_target' do
+      context 'empty inventory' do
+        let(:inventory) { Bolt::Inventory::Inventory2.new({}, config) }
+
+        it 'should parse a single target URI' do
+          name = 'notarget'
+          target = inventory.get_target(name)
+          expect(target.name).to eq(name)
+        end
+
+        it 'should error when an array of targets is requested' do
+          names = ['pcp://a', 'winrm://b', 'c']
+          expect {
+            inventory.get_target(names)
+          }.to raise_error(Bolt::Inventory::ValidationError, "'#{names}' refers to 3 targets")
+        end
+
+        it 'should fail for unknown protocols' do
+          expect {
+            inventory.get_target('z://foo')
+          }.to raise_error(Bolt::UnknownTransportError, %r{Unknown transport z found for z://foo})
+        end
+      end
+
+      context 'non-empty inventory' do
+        let(:inventory) {
```

Comment:
```suggestion
        let(:inventory) { Bolt::Inventory::Inventory2.new(data) }
```
And remove the 2 lines below

Reply:
This was copy pasted from line 299. Updated in both places.